service/service.py

```diff
@@ -75,24 +75,6 @@ def bad_request(error):
                    error='Bad Request',
                    message=message), status.HTTP_400_BAD_REQUEST
 
-@app.errorhandler(status.HTTP_404_NOT_FOUND)
-def not_found(error):
-    """ Handles resources not found with 404_NOT_FOUND """
-    message = str(error)
-    app.logger.warning(message)
-    return jsonify(status=status.HTTP_404_NOT_FOUND,
-                   error='Not Found',
-                   message=message), status.HTTP_404_NOT_FOUND
-
-@app.errorhandler(status.HTTP_500_INTERNAL_SERVER_ERROR)
-def internal_server_error(error):
-    """ Handles unexpected server error with 500_SERVER_ERROR """
-    message = str(error)
-    app.logger.error(message)
-    return jsonify(status=status.HTTP_500_INTERNAL_SERVER_ERROR,
-                   error='Internal Server Error',
-                   message=message), status.HTTP_500_INTERNAL_SERVER_ERROR
-
```

Comment:
Why are you removing these?

Reply:
They are not used since it looks like flask restplus handles all that now.
Earlier, these used to get covered in our tests and now this piece of code never gets executed.

Since all the APIs have moved over to flask restplus, there is no piece of code left that calls this